packages/nimble-components/src/table/components/row/styles.ts

```diff
@@ -9,19 +9,17 @@ import {
 } from '../../../theme-provider/design-tokens';
 
 export const styles = css`
-    ${display('flex')}
+    ${display('grid')}
```

Comment:
The `grid` layout more easily allows for the columns to maintain a particular width, even when their content extends beyond the bounds.